resources/js/features/games/components/GameAchievementSetsContainer/GameAchievementSet/SetRarityLabel/SetRarityLabel.tsx

```diff
@@ -33,13 +33,15 @@ export const SetRarityLabel: FC<SetRarityLabelProps> = ({ pointsTotal, pointsWei
         <p className="font-bold">{t('Rarity')}</p>
 
         <div className="flex flex-col gap-2">
```

Comment:
Verified this change works. Idea: can we lift `whitespace-normal` up to this div?